app/assets/javascripts/discourse/tests/acceptance/admin-site-settings-test.js

```diff
@@ -163,4 +166,30 @@ acceptance("Admin - Site Settings", function (needs) {
       "/admin/site_settings/category/all_results?filter=contact"
     );
   });
+
+  test("filters * and ? for domain lists", async (assert) => {
+    pretender.put("/admin/site_settings/blocked_onebox_domains", () => [200]);
```

Comment:
Instead of importing pretender like this, your tests should use the `needs.pretender` API. It makes sure we clean stuff up properly.

Reply:
Good to know, thank you.